app/views/exchange_rates/index_average_rates.html.erb

```diff
@@ -0,0 +1,52 @@
+<% content_for :top_breadcrumbs do %>
+   <%= generate_breadcrumbs "Yearly average rates",
+                            [ [t('breadcrumb.home'), home_path],
+                            [t('breadcrumb.tools'), tools_path],
+                            [t('breadcrumb.exchange_rates'), exchange_rates_path] ] %>
+
+ <% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= trade_tariff_heading %></span>
+    <h1 class="govuk-heading-l">HMRC Yearly average rates and <abbr title="value added tax">VAT</abbr></h1>
+    <p class="govuk-body-l">Check the official HMRC foreign currency exchange yearly average rates for Customs and <abbr title="value added tax">VAT</abbr>.</p>
+    <p class="govuk-body">Data is available in online (HTML), CSV and XML formats.</p>
+
+    <dt class="gem-c-metadata__term">From:</dt>
+    <dd class="gem-c-metadata__definition">
+      <strong><a href="https://www.gov.uk/government/organisations/hm-revenue-customs">HM Revenue &amp; Customs</a></strong>
+    </dd>
+    <dt class="gem-c-metadata__term">Published:</dt>
+    <dd class="gem-c-metadata__definition">
+      <%= '1 April 2011' %>
+    </dd>
+
+    <dt class="gem-c-metadata__term">Last updated:</dt>
+    <dd class="gem-c-metadata__definition">
+      <%= '3 May 2023' %>
+    </dd>
+
+    <h2>Documents</h2>
+
+    <% @period_file_list.each do |period_file| %>
+      <%= render partial: 'exchange_rates/document_average_rate_detail',
+                 locals: { end_date: period_file[:end_date], filename: period_file[:filename] } %>
+    <% end %>
+
+    <p>HMRC currency exchange rates for Customs and VAT are sourced from data provided by ft.com and xe.com.</p>
```

Comment:
I think we have removed this line now?

Reply:
Not sure, it is still there in the /exchange_rates
I am keeping it for now.

Reply:
this page will be removed soon, and merged with the exchange_rate/index.